docker/policies/add_instead_of_copy.rego

```diff
@@ -0,0 +1,36 @@
+package appshield.DS005
+
+__rego_metadata__ := {
+	"id": "DS005",
+	"title": "COPY Instead of ADD",
+	"version": "v1.0.0",
+	"severity": "MEDIUM",
+	"type": "Dockerfile Security Check",
+	"description": "Should use COPY instead of ADD unless, running a tar file",
+	"recommended_actions": "Replace ADD by COPY",
+	"url": "https://docs.docker.com/engine/reference/builder/#add",
+}
+
+__rego_input__ := {
+	"combine": "false",
+	"selector": [{"type": "dockerfile"}],
+}
+
+getAdd[args] {
```

Comment:
I know you align with the existing policies, but I think the snake case is natural in OPA.
https://www.openpolicyagent.org/docs/latest/

```suggestion
get_add[args] {
```